doc/manual/manual.tex

```diff
@@ -11868,13 +11868,13 @@ \subsubsection{Writing tests}
 
 To actually run the test, you have to go to your \aspect{} build directory and run
 \begin{lstlisting}[frame=single,language=ksh]
-    make setup\_tests
+    make setup_tests
 \end{lstlisting}
 so that your new test is added to the test suite. Then you can run it by executing
 \begin{lstlisting}[frame=single,language=ksh]
     ctest -R name_of_your_test -V
 \end{lstlisting}
-and you will get an output telling you if the test has
+in the \texttt{tests} folder and you will get an output telling you if the test has
```

Comment:
But you don't run the ctest command in the test folder, but in the build folder, right?

Reply:
so ... when I wrote this it did not work for me, only in the tests folder. Of course, now, fresh install, it works as intended in the build folder :(